x-pack/plugins/security_solution/public/detection_engine/rule_creation_ui/components/rule_preview/use_preview_rule.ts

```diff
@@ -12,13 +12,21 @@ import type {
   RuleCreateProps,
   RulePreviewResponse,
 } from '../../../../../common/api/detection_engine';
+import { METRIC_TYPE, TELEMETRY_EVENT, track } from '../../../../common/lib/telemetry';
 
 import { previewRule } from '../../../rule_management/api/api';
 import { transformOutput } from '../../../../detections/containers/detection_engine/rules/transforms';
 import type { TimeframePreviewOptions } from '../../../../detections/pages/detection_engine/rules/types';
 import { usePreviewInvocationCount } from './use_preview_invocation_count';
 import * as i18n from './translations';
 
+const trackPreview = (enableLoggedRequests: boolean) => {
+  track(METRIC_TYPE.COUNT, TELEMETRY_EVENT.PREVIEW_RULE_ALL);
```

Comment:
nit: maybe using event based telemetry here can be a bit simlier.

it will be just 1 event with property for enabled logs

Reply:
@nkhristinin 

Considered it, but went first with UI counters. On the second thought - it does not look to complicated to add EBT.
So, I reimplemented it